setup.py

```diff
@@ -75,6 +76,8 @@ def get_init_variable_value(var):
         lines = f.readlines()
     extras_require[extras_label] = [line.strip() for line in lines if line[0:2] != "-r"]
 
+# PEP517 workaround
+site.ENABLE_USER_SITE = True
 
```

Comment:
What does this do?

Reply:
Without it, I was getting "error: can't create or remove files in install directory" on `pip install -e .`. I tracked that down to https://www.scivision.dev/python-pip-devel-user-install/, I'll add the link to the comment.

Why are we installing scico in dev mode for the tests? I don't know.

Why were we not getting this error before? I don't know.

Reply:
I don't think there's a good reason for the `-e` for the tests, but I guess this is still useful if it's now necessary for enabling user-editable installation.